AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Task/FileAttachmentModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace AllReady.Areas.Admin.ViewModels.Task
```

Comment:
can we delete this entire class? I'm only asking b/c based on how it's being used, we might as well return `List<FileAttachment>` b/c the properties are the same and named the same